### PR TITLE
Add Chinese translation

### PR DESCRIPTION
**Pull request by KevinSJ**
Adding  Chinese translation

### CODE REVIEW

File: app/src/main/res/values-zh/strings.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    <string name="action_save">保存</string>
+    <string name="app_shortdesc">屏蔽广告和DNS服务器</string>
+    <string name="custom_dns">自定义DNS服务器</string>
+    <string name="dns_tab">DNS 服务器</string>
+    <string name="enable_hosts">启用域名过滤</string>
+    <string name="switch_onboot">开机时自动运行</string>
+    <string name="start_tab">启动/停止</string>
+    <string name="start_description">长按启动或关闭</string>
+    <string name="activity_edit">修改项目</string>
+    <string name="action">Regel</string>
```
Comment (julian-klode) [inline app/src/main/res/values-zh/strings.xml]: Untranslated, or rather the german translation.

File: app/src/main/res/values-zh/strings.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    <string name="location">位置 (URL 或 Host)</string>
+    <string name="title">标题</string>
+    <string-array name="item_states">
```
Comment (julian-klode) [inline app/src/main/res/values-zh/strings.xml]: Untranslated (same as above)

File: app/src/main/res/values-zh/strings.xml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    <string name="notification_stopping">正在停止</string>
+    <string name="notification_title">DNS66</string>
+    <string name="notification_waiting_for_net">等待网络</string>
+    <string name="action_refresh">更新Hosts文件</string>
+    <string name="load_defaults">载入初始化设置</string>
+    <string name="cannot_load_default_config">无法读取默认设置:%s - 这不应该发生</string>
+    <string name="hosts_tab">域名过滤器</string>
+    <string name="notification_reconnecting_error">重新连接错误</string>
+    <string name="info_app_license">This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.</string>
+    <![CDATA[
```
Comment (julian-klode) [inline app/src/main/res/values-zh/strings.xml]: Weird CDATA, what's up with that.

**Comment by julian-klode** [comment]
Oh, and I have to ask, while issue #6 is under discussion:

Would you be OK with your translation being included if we add some kind of exception to the license to be able to ship it on Amazon, as mentioned in issue #6?

**Comment by KevinSJ** [comment]
@julian-klode I will fix the untranslated part. I am fine with the license but I would be happy if you give me credit for the Chinese translation somewhere.

**Comment by KevinSJ** [comment]
I ll squash that.

**Comment by julian-klode** [comment]
No need to close the pull request, you can just push your squashed to your branch, and the pull request updates automatically. Opening a second one just pollutes things a bit, and gets rid of the previous discussion.

WRT translator credit: I don't have anything for that yet. I suspect I should add a translatable string with the name "translators" and each translation can list its translator there, and it then gets shown in the respective localized variant, so Chinese users see your name, German and English users see mine twice :/, and so on.

**Pull request closed without merging**